controller/resource_blackbox_test.go

```diff
@@ -188,21 +198,19 @@ func (rest *TestResourceREST) TestRegisterResourceWithParentResourceSetCreated()
 }
 
 func (rest *TestResourceREST) TestFailRegisterResourceUnknownOwner() {
-	resourceDescription := "Resource description"
 	resourceID := ""
 	resourceScopes := []string{}
 
 	// Attempt to register the resource with an unknown owner
 	resourceOwnerID := uuid.NewV4()
 
 	payload := &app.RegisterResourcePayload{
-		Description:      &resourceDescription,
 		Name:             "My new resource",
 		ParentResourceID: nil,
 		ResourceScopes:   resourceScopes,
 		ResourceID:       &resourceID,
 		ResourceOwnerID:  resourceOwnerID.String(),
-		Type:             "Area",
+		Type:             "openshift.io/resource/area",
 	}
 
 	test.RegisterResourceNotFound(rest.T(), rest.service.Context, rest.service, rest.securedController, payload)
```

Comment:
I don't see any tests which check Read action?

Reply:
TestRegisterResourceWithResourceIDSetCreated() tests read, also a new test in #249 tests read also.

Reply:
(see https://github.com/fabric8-services/fabric8-auth/pull/249/files#diff-4b125eae9f559a68ad43b9d1ae45cd18R220)

Reply:
Please add a test(s) for Read in this PR too.

Reply:
Done.